chart/governor-api/templates/configmap.yml

```diff
@@ -0,0 +1,55 @@
+{{- $root := . -}}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "common.names.fullname" . }}-config
+  annotations:
+    argocd.argoproj.io/sync-wave: '-1'
+data:
+  .governor.yaml: |
+    admin-groups:
+      - {{ .Values.governor.adminGroups }}
+
+    nats:
+      {{- if .Values.nats.url }}
+      url: {{ .Values.nats.url }}
+      {{- else }}
+      url: tls://governor-nats.{{ .Values.clusterInfo.fqdn }}:4222
```

Comment:
do we need to remove this? (not sure if `clusterInfo` is something we expose - i didnt see it in the other oss org for any active repos)

Reply:
good catch, thanks!